bin/rbenv-each

```diff
@@ -46,7 +71,13 @@ failed_rubies=""
 
 trap "exit 1" INT
 
-for ruby in $(rbenv versions --bare); do
+if [ -n "$aliases" ]; then
+    opts=
+else
+    opts="--skip-aliases"
```

Comment:
Right, that makes sense. Do you already have something in mind for how to do version detection?